test/integration/osl_mongodb/controls/osl_mongodb.rb

```diff
@@ -0,0 +1,64 @@
+control 'osl_mongodb' do
+  describe yum.repo('mongodb-org') do
+    it { should exist }
+    its('baseurl') { should cmp %r{https://repo\.mongodb\.org/yum/redhat/[0-9]+/mongodb-org/[0-9]+\.[0-9]/.+} }
+    it { should be_enabled }
+  end
+
+  describe package('mongodb-org') do
+    it { should be_installed }
+  end
+
+  describe file('/etc/mongod.conf') do
+    it { should exist }
+    its('owner') { should cmp 'root' }
+    its('group') { should cmp 'root' }
+    its('mode') { should cmp '0644' }
+    its('content') { should cmp /net:/ }
+    its('content') { should cmp /port: [0-9]+/ }
+    its('content') { should cmp /bindIp: [0-9]+\.[0-9]+\.[0-9]+.[0-9]+/ }
+    its('content') { should cmp /maxIncomingConnections: [0-9]+/ }
+    its('content') { should cmp /processManagement:/ }
+    its('content') { should cmp /fork: [false|true]/ }
+    its('content') { should cmp /pidFilePath: .+/ }
+    its('content') { should cmp /timeZoneInfo: .+/ }
+    its('content') { should cmp /storage:/ }
+    its('content') { should cmp /dbPath: .+/ }
+    its('content') { should cmp /journal:/ }
+    its('content') { should cmp /enabled: true/ }
+    its('content') { should cmp /systemLog:/ }
+    its('content') { should cmp /destination: .+/ }
+    its('content') { should cmp /logAppend: [false|true]/ }
+    its('content') { should cmp /path: .+/ }
```

Comment:
Since this file is relatively small, this might be a better way to test this:
```suggestion
    its('content') do
        should match <<~EOF.strip
          net:
            port: 27017
            bindIp: 127.0.0.1
            maxIncomingConnections: 65536
          processManagement:
            fork: true
            pidFilePath: /var/run/mongodb/mongod.pid
            timeZoneInfo: /usr/share/zoneinfo
          storage:
            dbPath: /var/lib/mongo
            journal:
              enabled: true
          systemLog:
            destination: file
            logAppend: true
            path: /var/log/mongodb/mongod.log
        EOF
    end
```